image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -1,26 +1,46 @@
-# OpenVINO Latent Consistency Model C++ image generation  pipeline
+# OpenVINO Latent Consistency Model C++ image generation pipeline with LoRA
+
 The pure C++ text-to-image pipeline, driven by the OpenVINO native API for SD v1.5 Latent Consistency Model with LCM Scheduler. It includes advanced features like LoRA integration with safetensors and [OpenVINO Tokenizers](https://github.com/openvinotoolkit/openvino_tokenizers). Loading `openvino_tokenizers` to `ov::Core` enables tokenization. [The common folder](../../common/) contains schedulers for image generation and `imwrite()` for saving `bmp` images. This demo has been tested for Linux platform only. There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/263-latent-consistency-models-image-generation/263-lcm-lora-controlnet.ipynb) which provides an example of image generaztion in Python.
 
 > [!NOTE]
 > This tutorial assumes that the current working directory is `<openvino.genai repo>/image_generation/lcm_dreamshaper_v7/cpp/` and all paths are relative to this folder.
 
 ## Step 1: Prepare build environment
 
-Prerequisites:
-- Conda ([installation guide](https://conda.io/projects/conda/en/latest/user-guide/install/index.html))
```

Comment:
why conda is not used?

Reply:
`conda install -c conda-forge openvino` will automatically print and set env var with `SET "VS_YEAR=2019"`, while only VS2022 exists on my Win11.
The details in the attachment. [log.txt](https://github.com/user-attachments/files/15788970/log.txt)
Shall we align with the previous text generation without conda?